### PR TITLE
upgrade to monaco-editor 0.49.0

### PR DESCRIPTION
**Pull request by morgilad**
Upgrade to monaco-editor 0.49.0

### CODE REVIEW

**Review by Weffe — DISMISSED** [review]
LGTM

File: yarn.lock [hunk 1]
```diff
@@ -8034,17 +8034,10 @@ __metadata:
...
   linkType: hard
 
-"monaco-editor@npm:0.46.0":
-  version: 0.46.0
-  resolution: "monaco-editor@npm:0.46.0"
-  checksum: be7aaa2747b8d3a3537f56c9353ebe779b188eb1480f6453fa1e346d8ccdd27068cb4508c8f6f3b8cc07ac0f792a3d6df36168d54cd6e074742182525115be86
-  languageName: node
-  linkType: hard
-
-"monaco-editor@patch:monaco-editor@npm%3A0.46.0#./.yarn/patches/monaco-editor-npm-0.46.0-fb69b10c11.patch::locator=monaco-kusto%40workspace%3A.":
```
Comment (maxburs) [inline yarn.lock:8044]: Patch needs to be updated?

Either:
- Patch is still required, and we need to update it. This also implies the CI script to check if the samples work is broken 😞
- Patch is no longer required, and we can delete it! 🎊

Comment (morgilad) [reply yarn.lock:8044]: What issue does this patch resolve? How can I verify it's working as expected? 
I checked and saw that my local environment is functioning well, but I haven't tested it locally on KustoWeb or Trident due to the linking issue we discussed. 
Additionally, if the 'test samples' action passed, doesn't that indicate it's not broken?

Comment (morgilad) [reply yarn.lock:8044]: I’ve updated the patch file. 
However, I’m still uncertain why the samples test doesn’t result in a failure on GitHub Actions.
@maxburs waiting for your response before merging to master.

Comment (maxburs) [reply yarn.lock:8044]: > What issue does this patch resolve? How can I verify it's working as expected?

The Parcel samples will be broken if the patch isn't applied and is still necessary. The patch contains a link to the relevant Parcel bug, and the samples README's contain more details.

> However, I’m still uncertain why the samples test doesn’t result in a failure on GitHub Actions.

I'd guess there's a mistake in the script. I had a lot of problems trying to get it to start/stop each sample. Sometimes, a previous sample wouldn't stop, and the script would re-test the same sample multiple times.

Comment (maxburs) [reply yarn.lock:8044]: Problem spot is here:

/scripts/test-samples.ts
image

**Pull request merged**